.github/workflows/installations.yml

```diff
@@ -1,47 +1,39 @@
 ---
-
 name: Demultiplex installation tests
 
 "on": ["pull_request"]
 
 jobs:
-
-  installations:
-
-    runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        python-version: [3.6, 3.7]
-
-    steps:
-
-    - name: Check out git repository
-      uses: actions/checkout@v2
-
-    # Set up python
-    - name: Set up Python ${{ matrix.python-version}}
-      uses: actions/setup-python@v2
-      with:
-        python-version: ${{ matrix.python-version}}
-
-    - name: Install pip
-      # Make sure that we have correct python version and pip version
-      # For details see .github/workflows/server_info.md
-      run: |
-        if [ "$PYTHON_VERSION" = "$OLD_VERSION" ]; then
-          python -m pip install --upgrade pip==9.0.1
-        else
-          python -m pip install --upgrade pip==18.1
-        fi
-        echo $PYTHON_VERSION
-        pip --version
-      env:
-        PYTHON_VERSION: ${{ matrix.python-version}}
-        OLD_VERSION: 3.6
-
-    - name: Production Dependencies
-      run: |
-        pip install micropipenv[toml]
-        micropipenv requirements --method poetry > requirements.txt
-        pip install -r requirements.txt
-        pip check
+    installations:
+        runs-on: ubuntu-latest
+        strategy:
+            matrix:
+                python-version: [3.7]
+
+        steps:
+            - name: Check out git repository
+              uses: actions/checkout@v2
+
+            # Set up python
+            - name: Set up Python ${{matrix.python-version}}
+              uses: actions/setup-python@v2
+              with:
+                  python-version: ${{matrix.python-version}}
+
+            - name: Install pip
+              # Make sure that we have correct python version and pip version
+              # For details see .github/workflows/server_info.md
+              run: |
+                  python -m pip install --upgrade pip==18.1
+                  echo $PYTHON_VERSION
+                  pip --version
+              env:
+                  PYTHON_VERSION: ${{matrix.python-version}}
+                  OLD_VERSION: 3.6
```

Comment:
```suggestion
```
Can't this line just be removed as we won't be using this version of python anymore (if I interpreted it correctly)?